build-system/tasks/runtime-test/status-report.js

```diff
@@ -37,8 +37,8 @@ const TEST_TYPE_SUBTYPES = new Map([
   // TODO(danielrozenberg): add 'e2e' tests.
 ]);
 const TEST_TYPE_BUILD_TARGETS = new Map([
-  ['integration', ['RUNTIME', 'BUILD_SYSTEM', 'INTEGRATION_TEST']],
-  ['unit', ['RUNTIME', 'BUILD_SYSTEM', 'UNIT_TEST']],
+  ['integration', ['RUNTIME', 'FLAG_CONFIG', 'INTEGRATION_TEST']],
+  ['unit', ['RUNTIME', 'UNIT_TEST']],
```

Comment:
Q: are unit tests never affected by flag configs?

Reply:
Maybe worth putting that question in the slack/team chat

Reply:
> Q: are unit tests never affected by flag configs?

A: Unit tests are unaffected by flag configs.

Flag configs are only used in compiled runtimes (`v0.js`, `shadow-v0.js`, etc.). Unit tests do not run against compiled runtime code.

The only tests that use the compiled runtime are integration tests (minified), E2E tests (unminified), and visual diff (unminified). (Side note: we should run E2E and visual diff against the minified runtime.)